R/transformCounts.R

```diff
@@ -56,6 +56,9 @@
 #' 'relabundance', 'rrank', 'standardize', 'total': please refer to 
 #' \code{\link[vegan:decostand]{decostand}} for details.
 #' 
+#' \item 'css': Cumulative Sum Scaling (CSS) can be used to normalize count data by accounting for differences 
+#' in library sizes.
```

Comment:
Any chance to include a literature reference where this method is explained? 

Also explain that this requires the cutoff parameter (default 0.75)